peachjam_api/views.py

```diff
@@ -29,6 +31,24 @@ class RelationshipViewSet(viewsets.ModelViewSet):
     serializer_class = RelationshipSerializer
 
 
+class AnnotationViewSet(viewsets.ModelViewSet):
+    queryset = Annotation.objects.all()
+    serializer_class = AnnotationSerializer
```

Comment:
It's good to always be explicit about permissions for these views. In this case, DjangoModelPermissions should suffice - it enforces authentication.